### ISSUE TITLE
Team / multiple authors support

### ISSUE THREAD

**Issue by jacksingleton** [sandstorm-io/sandstorm#1607]
In the app market, the SandForms app has me listed as the author, and links to my twitter and github account using my gpg signed statement and keybase.io

However, SandForms is a team effort and I would love to credit the team while still retaining the verified links to our social media accounts.

**Comment by ocdtrekkie** [comment]
Coukd you not create a GPG key for your organization, and make a Keybase for it, and verify it with said statements?

**Comment by jacksingleton** [comment]
ah that could work we'll give it a go, thanks

**Comment by kentonv** [comment]
Yes, I think the right solution here is to create an identity for the team.

Of course, while creating a new twitter account is straightforward, for Github it would be better to use an organization (and on G+ and Facebook, if they are ever supported, you'd want to use a "page"). In order to support any of that, we really need Keybase to support it. There are various Keybase issues about this but not clear if any progress is occurring:

https://github.com/keybase/keybase-issues/issues/330
https://github.com/keybase/keybase-issues/issues/783
https://github.com/keybase/keybase-issues/issues/1983
